pkg/platform/model/vcs.go

```diff
@@ -989,7 +997,11 @@ func GetCommit(commitID strfmt.UUID) (*mono_models.Commit, error) {
 	params.SetCommitID(commitID)
 	params.SetHTTPClient(retryhttp.DefaultClient.StandardClient())
 
-	res, err := authentication.Client().VersionControl.GetCommit(params, authentication.ClientAuth())
+	client := mono.New()
+	if authentication.LegacyGet().Authenticated() {
```

Comment:
This is not ideal, but necessary in order to avoid authentication being required to perform a revert. This `LegacyGet()` is used elsewhere throughout this file in similar places. It just was never added here when it should have been.

I could get rid of it by passing `*authentication.Auth` pointers to this function, but that would affect other unrelated callers like *internal/runners/pull/pull.go*. We have a ticket to refactor out `LegacyGet()`.